SofaKernel/framework/sofa/core/VecId.h

```diff
@@ -150,8 +151,14 @@ class TStandardVec<V_MATDERIV, vaccess>
 public:
     typedef TVecId<V_MATDERIV, vaccess> MyVecId;
 
-    static MyVecId holonomicC()    { return MyVecId(1);}
-    static MyVecId nonHolonomicC() { return MyVecId(2);}
+    static MyVecId constraintMatrix()    { return MyVecId(1);} // jacobian matrix of constraints
```

Comment:
The matrix names are really unclear. Would it be possible to have some doc about it?
'constraintMatrix' seems to be 'constraintJacobian'? Does that mean that only the mstates in the same node as a Constraint would have such a matrix?
Whereas 'mappingMatrix' should be 'sparseJacobian', and every mstates should got one?